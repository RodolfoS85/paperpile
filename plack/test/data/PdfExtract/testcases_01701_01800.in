---
file: ../../../PDFREPOSITORY/001701.pdf
---
file: ../../../PDFREPOSITORY/001702.pdf
---
file: ../../../PDFREPOSITORY/001703.pdf
---
file: ../../../PDFREPOSITORY/001704.pdf
---
file: ../../../PDFREPOSITORY/001705.pdf
---
file: ../../../PDFREPOSITORY/001706.pdf
---
file: ../../../PDFREPOSITORY/001707.pdf
---
file: ../../../PDFREPOSITORY/001708.pdf
---
file: ../../../PDFREPOSITORY/001709.pdf
---
file: ../../../PDFREPOSITORY/001710.pdf
---
file: ../../../PDFREPOSITORY/001711.pdf
---
file: ../../../PDFREPOSITORY/001712.pdf
---
file: ../../../PDFREPOSITORY/001713.pdf
---
file: ../../../PDFREPOSITORY/001714.pdf
---
file: ../../../PDFREPOSITORY/001715.pdf
---
file: ../../../PDFREPOSITORY/001716.pdf
---
file: ../../../PDFREPOSITORY/001717.pdf
---
file: ../../../PDFREPOSITORY/001718.pdf
---
file: ../../../PDFREPOSITORY/001719.pdf
---
file: ../../../PDFREPOSITORY/001720.pdf
---
file: ../../../PDFREPOSITORY/001721.pdf
---
file: ../../../PDFREPOSITORY/001722.pdf
---
file: ../../../PDFREPOSITORY/001723.pdf
---
file: ../../../PDFREPOSITORY/001724.pdf
---
file: ../../../PDFREPOSITORY/001725.pdf
---
file: ../../../PDFREPOSITORY/001726.pdf
---
file: ../../../PDFREPOSITORY/001727.pdf
---
file: ../../../PDFREPOSITORY/001728.pdf
---
file: ../../../PDFREPOSITORY/001729.pdf
---
file: ../../../PDFREPOSITORY/001730.pdf
---
file: ../../../PDFREPOSITORY/001731.pdf
---
file: ../../../PDFREPOSITORY/001732.pdf
---
file: ../../../PDFREPOSITORY/001733.pdf
---
file: ../../../PDFREPOSITORY/001734.pdf
---
file: ../../../PDFREPOSITORY/001735.pdf
---
file: ../../../PDFREPOSITORY/001736.pdf
---
file: ../../../PDFREPOSITORY/001737.pdf
---
file: ../../../PDFREPOSITORY/001738.pdf
---
file: ../../../PDFREPOSITORY/001739.pdf
---
file: ../../../PDFREPOSITORY/001740.pdf
---
file: ../../../PDFREPOSITORY/001741.pdf
---
file: ../../../PDFREPOSITORY/001742.pdf
---
file: ../../../PDFREPOSITORY/001743.pdf
---
file: ../../../PDFREPOSITORY/001744.pdf
---
file: ../../../PDFREPOSITORY/001745.pdf
---
file: ../../../PDFREPOSITORY/001746.pdf
---
file: ../../../PDFREPOSITORY/001747.pdf
---
file: ../../../PDFREPOSITORY/001748.pdf
---
file: ../../../PDFREPOSITORY/001749.pdf
---
file: ../../../PDFREPOSITORY/001750.pdf
---
file: ../../../PDFREPOSITORY/001751.pdf
---
file: ../../../PDFREPOSITORY/001752.pdf
---
file: ../../../PDFREPOSITORY/001753.pdf
---
file: ../../../PDFREPOSITORY/001754.pdf
---
file: ../../../PDFREPOSITORY/001755.pdf
---
file: ../../../PDFREPOSITORY/001756.pdf
---
file: ../../../PDFREPOSITORY/001757.pdf
---
file: ../../../PDFREPOSITORY/001758.pdf
---
file: ../../../PDFREPOSITORY/001759.pdf
---
file: ../../../PDFREPOSITORY/001760.pdf
---
file: ../../../PDFREPOSITORY/001761.pdf
---
file: ../../../PDFREPOSITORY/001762.pdf
---
file: ../../../PDFREPOSITORY/001763.pdf
---
file: ../../../PDFREPOSITORY/001764.pdf
---
file: ../../../PDFREPOSITORY/001765.pdf
---
file: ../../../PDFREPOSITORY/001766.pdf
---
file: ../../../PDFREPOSITORY/001767.pdf
---
file: ../../../PDFREPOSITORY/001768.pdf
---
file: ../../../PDFREPOSITORY/001769.pdf
---
file: ../../../PDFREPOSITORY/001770.pdf
---
file: ../../../PDFREPOSITORY/001771.pdf
---
file: ../../../PDFREPOSITORY/001772.pdf
---
file: ../../../PDFREPOSITORY/001773.pdf
---
file: ../../../PDFREPOSITORY/001774.pdf
---
file: ../../../PDFREPOSITORY/001775.pdf
---
file: ../../../PDFREPOSITORY/001776.pdf
---
file: ../../../PDFREPOSITORY/001777.pdf
---
file: ../../../PDFREPOSITORY/001778.pdf
---
file: ../../../PDFREPOSITORY/001779.pdf
---
file: ../../../PDFREPOSITORY/001780.pdf
---
file: ../../../PDFREPOSITORY/001781.pdf
---
file: ../../../PDFREPOSITORY/001782.pdf
---
file: ../../../PDFREPOSITORY/001783.pdf
---
file: ../../../PDFREPOSITORY/001784.pdf
---
file: ../../../PDFREPOSITORY/001785.pdf
---
file: ../../../PDFREPOSITORY/001786.pdf
---
file: ../../../PDFREPOSITORY/001787.pdf
---
file: ../../../PDFREPOSITORY/001788.pdf
---
file: ../../../PDFREPOSITORY/001789.pdf
---
file: ../../../PDFREPOSITORY/001790.pdf
---
file: ../../../PDFREPOSITORY/001791.pdf
---
file: ../../../PDFREPOSITORY/001792.pdf
---
file: ../../../PDFREPOSITORY/001793.pdf
---
file: ../../../PDFREPOSITORY/001794.pdf
---
file: ../../../PDFREPOSITORY/001795.pdf
---
file: ../../../PDFREPOSITORY/001796.pdf
---
file: ../../../PDFREPOSITORY/001797.pdf
---
file: ../../../PDFREPOSITORY/001798.pdf
---
file: ../../../PDFREPOSITORY/001799.pdf
---
file: ../../../PDFREPOSITORY/001800.pdf
