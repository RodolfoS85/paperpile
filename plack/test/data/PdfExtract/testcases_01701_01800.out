---
authors: 'Haag-Liautard, Cathy and Coffey, Nicole and Houle, David and Lynch, Michael and Charlesworth, Brian and Keightley, Peter D.'
doi: 10.1371/journal.pbio.0060204
title: Direct Estimation of the Mitochondrial DNA Mutation Rate in Drosophila melanogaster
---
authors: 'Petruk, Svetlana and Sedkov, Yurii and Brock, Hugh W. and Mazo, Alexander'
title: A Model for Initiation of Mosaic HOX Gene Expression Patterns by Non-Coding RNAs in Early Embryos
---
authors: 'Fang, Jie and Zhang, Bei and Chen, Ning and Asai, Hiroshi'
title: Chemical Modification of Glycerinated Stalks Shows Tyrosine Residues Essential for Spasmoneme Contraction of Vorticella sp
---
authors: 'Sato, Jun J. and Wolsan, Mieczyslaw and Suzuki, Hitoshi and Hosoda, Tetsuji and Yamaguchi, Yasunori and Hiyama, Kozue and Kobayashi, Mari and Minami, Shinji'
doi: 10.2108/zsj.23.125
title: 'Evidence from Nuclear DNA Sequences Sheds Light on the Phylogenetic Relationships of Pinnipedia: Single Origin with Affinity to Musteloidea'
---
authors: 'Lesh, Neal and Mitzenmacher, Michael'
title: A Complete and Effective Move Set for Simplified Protein Folding
---
authors: 'Li, Tao and Zhu, Shenghuo and Li, Qi and Ogihara, Mitsunori'
title: Gene Functional classification by Semi-supervised Learning from Heterogeneous Data
---
authors: 'Livny, Jonathan and Waldor, Matthew K'
doi: 10.1016/j.mib.2007.03.005
title: Identification of small RNAs in diverse bacterial species
---
authors: 'Loots, G. G. and Locksley, R. M. and Blankespoor, C. M. and Wang, Z. E. and Miller, W. and Rubin, E. M. and Frazer, K. A.'
title: 'Identification of a Coordinate Regulator of Interleukins 4, 13, and 5 by Cross-Species Sequence Comparisons'
---
authors: 'Lukashin, Alexander V. and Fuchs, Rainer'
title: 'Analysis of temporal gene expression profiles: clustering by simulated annealing and determining the optimal number of clusters'
---
authors: 'Mandin, Pierre and Gottesman, Susan'
doi: 10.1111/j.1365-2958.2009.06665.x
title: A genetic approach for finding small RNAs regulators of genes of interest identifies RybC as regulating the DpiA/DpiB two-component system
---
authors: 'Mann, Martin and Hamra, Mohamed Abou and Steinhfel, Kathleen and Backofen, Rolf'
title: Constraint-based Local Move Definitions for Lattice Protein Models Including Side Chains
---
arxivid: 0910.2559v1
authors: 'Mann, Martin'
title: Efficient exploration of energy landscapes
---
authors: 'Marchini, Jonathan and Donnelly, Peter and Cardon, Lon R'
doi: 10.1038/ng1537
title: Genome-wide strategies for detecting multiple loci that influence complex diseases
---
authors: 'Markovtsov, Vadim and Nikolic, Julia M. and Goldman, Joseph A. and Turck, Christoph W. and Chou, MIN-YUAN and Black, Douglas L.'
title: Cooperative Assembly of an hnRNP Complex Induced by a Tissue-Specific Homolog of Polypyrimidine Tract Binding Protein
---
authors: 'Mauri, Giancarlo and Pavesi, Giulio'
title: Pattern Discovery in RNA Secondary Structure Using Affix Trees
---
authors: 'Meyer, Carsten and Giegerich, Robert'
title: Matching and Significance Evaluation of combined Sequence-Structure Motifs in RNA
---
authors: 'Miles, Amanda K. and Matharoo-Ball, Balwir and Li, Geng and Ahmad, Murrium and Rees, Robert C.'
doi: 10.1007/s00262-005-0115-5
title: 'The identification of human tumour antigens: current status and future developments'
---
authors: 'Model, Fabian and Adorjan, Peter and Olek, Alexander and Piepenbrock, Christian'
title: Feature selection for DNA methylation based cancer classification
---
arxivid: cond-mat/0206225
authors: 'Mller, M.'
title: Statistical Physics of RNA-folding
---
authors: 'Nielsen, Henrik and Engelbrecht, Jacob and Brunak, Søren and von Heijne, Gunnar'
title: Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites
---
authors: 'Niu, Qi-Wen and Lin, Shih-Shun and Reyes, Jose Luis and Chen, Kuan-Chun and Wu, Hui-Wen and Yeh, Shyi-Dong and Chua, Nam-Hai'
doi: 10.1038/nbt1255
title: Expression of artificial microRNAs in transgenic Arabidopsis thaliana confers virus resistance
---
authors: 'Olivier, Catherine and Poirier, Guillaume and Gendron, Patrick and Boisgontier, Anita and Major, Francois and Chartrand, Pascal'
doi: 10.1128/MCB.25.11.4752-4766.2005
title: Identification of a Conserved RNA Motif Essential for She2p Recognition and mRNA Localization to the Yeast Bud
---
authors: 'Oumard, A. and Hennecke, M. and Hauser, H. and Nourbakhsh, M.'
title: Translation of NRF mRNA Is Mediated by Highly Efficient Internal Ribosome Entry
---
authors: 'Parida, Laxmi and Rigoutsos, Isidore and Floratos, Aris and Platt, Dan and Gao, Yuan'
title: 'Pattern Discovery on Character Sets and Real-valued Data: Linear Bound on Irredundant Motifs and an Efficient Polynomial Time Algorithm'
---
authors: 'Pavesi, Giulio and Mauri, Giancarlo and Pesole, Graziano'
title: An algorithm for finding signals of unknown length in DNA sequences
---
authors: 'Pavlidis, Paul and Weston, Jason'
title: Gene functional classification from heterogeneous data
---
authors: 'Pesole, Graziano and Liuni, Sabino and DSouza, Mark'
title: 'PatSearch: a pattern matcher software that finds functional elements in nucleotide and protein sequences and assesses their statistical significance'
---
authors: 'Pfeifer, Henning and Conrad, Marcus and Roethlein, Doris and Kyriakopoulos, Antonios and Brielmeier, Markus and Bornkamm, Georg W. and Behne, Dietrich'
doi: 10.1096/fj.00-0655fje
title: 'Identification of a specific sperm nuclei selenoenzyme necessary for protamine thiol cross- linking during sperm maturation'
---
authors: 'Ponomarenko, Julia V. and Bourne, Philip E. and Shindyalov, Ilya N.'
title: Building an automated classification of DNA-binding protein domains
---
authors: 'Akutsu, Tatsuya and Miyano, Satoru'
title: On the approximation of protein threading'
---
authors: 'Auffermann, William F. and Ngan, Shing-Chung and Hu, Xiaoping'
doi: 10.1006/nimg.2002.1223
title: Cluster Significance Testing Using the Bootstrap
---
authors: 'Backofen, Rolf and Tsur, Dekel and Zakov, Shay and Ziv-Ukelson, Michal'
title: 'Sparse RNA Folding: Time and Space Efficient Algorithms'
---
authors: 'Balasubramanian, Vijay and Berry II, Michael J. and Kimber, Don'
title: Metabolically Efficient Information Processing
---
authors: 'Baldi, Pierre and Brunak, Søren and Chauvin, Yves and Andersen, Claus A. F. and Nielsen, Henrik'
title: 'Assessing the accuracy of prediction algorithms for classification: an overview'
---
authors: 'Bar-Noy, Shoshana and Moskovitz, Jackob'
title: 'Mouse methionine sulfoxide reductase B: effect of selenocysteine incorporation on its activity and expression of the seleno-containing enzyme in bacterial and mammalian cells'
---
authors: 'Baraniak, Andrew P. and Lasda, Erika L. and Wagner, Eric J. and Garcia-Blanco, Mariano A.'
doi: 10.1128/MCB.23.24.9327-9337.2003
title: A Stem Structure in Fibroblast Growth Factor Receptor 2 Transcripts Mediates Cell-Type-Specific Splicing by Approximating Intronic Control Elements
---
authors: 'Becker, A. and Kaufmann, K. and Freialdenhoven, A. and Vincent, C. and -A. Li, M. and Saedler, H. and Theissen, G.'
doi: 10.1007/s00438-001-0615-8
title: A novel MADS-box gene subfamily with a sister-group relationship to class B floral homeotic genes
---
authors: 'Bejerano, Gill'
title: Efficient Exact p -Value Computation and Applications to Biosequence Analysis
---
authors: 'Belting, Heinz-Georg and Hauptmann, Giselbert and Meyer, Dirk and Abdelilah-Seyfried, Salim and Chitnis, Ajay and Eschbach, Cathrin and Söll, Iris and Thisse, Christine and Thisse, Bernard and Artinger, Kristin B. and Lunde, Karen and Driever, Wolfgang'
title: spiel ohne grenzen/pou2 is required during establishment of the zebrafish midbrain-hindbrain boundary organizer
---
authors: 'Bentwich, Isaac and Avniel, Amir and Karov, Yael and Aharonov, Ranit and Gilad, Shlomit and Barad, Omer and Barzilai, Adi and Einat, Paz and Einav, Uri and Meiri, Eti and Sharon, Eilon and Spector, Yael and Bentwich, Zvi'
doi: 10.1038/ng1590
title: Identification of hundreds of conserved and nonconserved human microRNAs
---
authors: 'Bilodeau, Patricia S. and Domsic, Jeffrey K. and MARTIN STOLTZFUS, C.'
title: Splicing Regulatory Elements within tat Exon 2 of Human Immunodeficiency Virus Type 1 (HIV-1) Are Characteristic of Group M but Not Group O HIV-1 Strains
---
authors: 'Breimer, Eric and Goldberg, Mark'
title: 'Learning Significant Alignments: An Alternative to Normalized Local Alignment'
---
authors: 'Buchner, David A. and Trudeau, Michelle and Meisler, Miriam H.'
title: 'SCNM1, a Putative RNA Splicing Factor That Modifies Disease Severity in Mice'
---
authors: 'Buckanovich, Ronald J. and Darnell, Robert B.'
title: The Neuronal RNA Binding Protein Nova-1 Recognizes Specific RNA Targets In Vitro and In Vivo
---
authors: 'Buratti, Emanuele and Baralle, Francisco E.'
doi: 10.1128/MCB.24.24.10505-10514.2004
title: Influence of RNA Secondary Structure on the Pre-mRNA Splicing Process
---
authors: 'Chan, Raymond C. and Black, Douglas L.'
title: The Polypyrimidine Tract Binding Protein Binds Upstream of Neural Cell-Specific c-src Exon N1 To Repress the Splicing of the Intron Downstream
---
authors: 'Chinnasamy, A. and Sung, W. K.'
title: PROTEIN STRUCTURE AND FOLD PREDICTION USING TREE-AUGMENTED NAIVE BAYESIAN CLASSIFIER
---
authors: 'Chow, M. L. and Moler, E. J. and Mian, I. S.'
title: Identifying marker genes in transcription profiling data using a mixture of feature relevance experts
---
authors: 'Christiansen, Henning and Have, Christian Theil and Lassen, Ole Torp and Petit, Matthieu'
title: 'A Constraint Model for Constrained Hidden Markov Models: a first Biological Application'
---
authors: 'Coghlan, Avril and Mac Donaill, Donall A. and Buttimore, Nigel H.'
title: Representation of amino acids as five-bit or three-bit patterns for filtering protein databases
---
authors: 'Cowperthwaite, Matthew C. and Bull, J. J. and Meyers, Lauren Ancel'
doi: 10.1534/genetics.104.039248
title: Distributions of Beneficial Fitness Effects in RNA
---
authors: 'Cohen, Ira and Sebe, Nicu and Cozman, Fabio G. and Cirelo, Marcelo C. and Huang, Thomas S.'
title: Learning Bayesian network classifiers for facial expression recognition using both labeled and unlabeled data
---
authors: 'Cui, Yan and Wong, Wing Hung'
title: Multiple-Sequence Information Provides Protection against Mis-Specified Potential Energy Functions in the Lattice Model of Proteins
---
authors: 'Dash, M. and Liu, H.'
title: Feature Selection for Classification
---
authors: 'Davuluri, Ramana V. and Grosse, Ivo and Zhang, Michael Q.'
doi: 10.1038/ng780
title: Computational identification of promoters and first exons in the human genome
---
authors: 'De Moerlooze, Laurence and Spencer-Dene, Bradley and Revest, Jean-Michel and Hajihosseini, Mohammad and Rosewell, Ian and Dickson, Clive'
title: An important role for the IIIb isoform of fibroblast growth factor receptor 2 (FGFR2) in mesenchymal-epithelial signalling during mouse organogenesis
---
authors: 'Diaz, Naryttza N and Krause, Lutz and Goesmann, Alexander and Niehaus, Karsten and Nattkemper, Tim W'
doi: 10.1186/1471-2105-10-56
title: 'TACOA - Taxonomic classification of environmental genomic fragments using a kernelized nearest neighbor approach'
---
authors: 'Dror, Gideon and Sorek, Rotem and Shamir, Ron'
title: Accurate identification of alternatively spliced exons using support vector machine
---
authors: 'Faeder, J. R. and Blinov, M. L. and Goldstein, B. and Hlavacek, W. S.'
doi: 10.1049/sb:20045031
title: Combinatorial complexity and dynamical restriction of network flows in signal transduction
---
authors: 'Fairbrother, William G. and Yeh, Ru-Fang and Sharp, Phillip A. and Burge, Christopher B.'
title: Predictive Identification of Exonic Splicing Enhancers in Human Genes
---
authors: 'Filikov, Anton V. and Mohan, Venkatraman and Vickers, Timothy A. and Griffey, Richard H. and Dan Cook, P. and Abagyan, Ruben A. and James, Thomas L.'
title: 'Identification of ligands for RNA targets via structure-based virtual screening: HIV-1 TAR'
---
authors: 'Boutilier, Craig and Friedman, Nir and Goldszmidt, Moises and Koller, Daphne'
title: Context-Specific Independence in Bayesian Networks
---
authors: 'Furey, Terrence S. and Cristianini, Nello and Duffy, Nigel and Bednarski, David W. and Schummer, Michel and Haussler, David'
title: Support vector machine classification and validation of cancer tissue samples using microarray expression data
---
authors: 'Geetha, V. and Di Francesco, Valentina and Garnier, Jean and Munson, Peter J.'
title: Comparing protein sequence-based and predicted secondary structure-based methods for identification of remote homologs
---
authors: 'Ghisolfi-Nieto, Laurence and Joseph, Gerard and Puvion-Dutilleul, Francine and Amalric, Francois and Bouvet, Philippe'
title: 'Nucleolin is a Sequence-specific RNA-binding Protein: Characterization of Targets on Pre-ribosomal RNA'
---
authors: 'Golub, T. R. and Slonim, D. K. and Tamayo, P. and Huard, C. and Gaasenbeek, M. and Mesirov, J. P. and Coller, H. and Loh, M. L. and Downing, J. R. and Caligiuri, M. A. and Bloomfield, C. D. and Lander, E. S.'
title: 'Molecular Classification of Cancer: Class Discovery and Class Prediction by Gene Expression Monitoring'
---
authors: 'Gorodkin, J.'
doi: 10.1016/j.compbiolchem.2004.09.006
title: Comparing two K-category assignments by a K-category correlation coefficient
---
authors: 'Grabowski, Robert and Kuhlmann, Marco and Mhl, Mathias'
title: Lexicalised Configuration Grammars
---
authors: 'Grayson, Jason and SANDERS WILLIAMS, R. and Yu, YIE-TEH'
title: Synergistic Interactions between Heterologous Upstream Activation Elements and Specific TATA Sequences in a Muscle-Specific Promoter
---
authors: 'Han, Hogyu and Dervan, Peter'
title: Sequence-specifi'c recognition of double helical RNA and RNADNA by triple helix formation
---
authors: 'Heitsch, Christine E.'
title: The Global Arrangement of Local Reconfigurations under an RNA Secondary Structure Design Procedure
---
authors: 'Hertel, Jana and Hofacker, Ivo L. and Stadler, Peter F.'
title: 'SnoReport: Computational identification of snoRNAs with unknown targets'
---
authors: 'Hohl, Michael and Kurtz, Stefan and Ohlebusch, Enno'
title: Efficient multiple genome alignment
---
authors: 'Hsu, Chih-Wei and Chang, Chih-Chung and Lin, Chih-Jen'
title: A Practical Guide to Support Vector Classification
---
authors: 'Hsu, Eric I. and McIlraith, Sheila A.'
title: 'Statement of Ongoing Work: Extending Boolean Satisfiability Techniques for Haplotype Inference by Pure Parsimony'
---
authors: 'Jager, Dominik and Sharma, Cynthia M. and Thomsen, Jens and Ehlers, Claudia and Vogel, Jorg and Schmitz, Ruth A.'
doi: 10.1073/pnas.0909051106
title: Deep sequencing analysis of the Methanosarcina mazei Go1 transcriptome in response to nitrogen availability
---
authors: 'Jansson, Jesper and Ng, See-Kiong and Sung, Wing-Kin and Willy, Hugo'
title: A Faster and More Space-Efficient Algorithm for Inferring Arc-Annotations of RNA Sequences Through Alignment
---
authors: 'Jansson, Jesper and Ng, See-Kiong and Sung, Wing-Kin and Willy, Hugo'
doi: 10.1007/s00453-006-1207-0
title: A Faster and More Space-Efficient Algorithm for Inferring Arc-Annotations of RNA Sequences through Alignment
---
authors: 'Jensen, L. J. and Gupta, R. and Blom, N. and Devos, D. and Tamames, J. and Kesmir, C. and Nielsen, H. and Stærfeldt, H. H. and Rapacki, K. and Workman, C. and Andersen, C. A. F. and Knudsen, S. and Krogh, A. and Valencia, A. and Brunak, S.'
doi: 10.1016/S0022-28360200379-0
title: Prediction of Human Protein Function from Post-translational Modifications and Localization Features
---
authors: 'Kanemaki, Masato and Sanchez-Diaz, Alberto and Gambus, Agnieszka and Labib, Karim'
title: Functional proteomic identification of DNA replication proteins by induced proteolysis in vivo
---
authors: 'Keich, U. and Pevzner, P. A.'
title: 'Subtle motifs: defining the limits of motif finding algorithms'
---
authors: 'Kim, Dong Kyue and Sim, Jeong Seop and Park, Heejin and Park, Kunsoo'
title: Linear-Time Construction of Suffix Arrays
---
authors: 'Klymkowsky, Michael W. and Garvin-Doxas, Kathy and Zeilik, Michael'
doi: 10.1187/cbe.03-03-0014
title: 'Bioliteracy and Teaching Efficacy: What Biologists Can Learn from Physicists'
---
authors: 'Ko, Pang and Aluru, Srinivas'
title: Space Efficient Linear Time Construction of Suffix Arrays
---
authors: 'Kromayer, M. and Wilting, R. and Tormay, P. and Bock, A.'
title: Domain Structure of the Prokaryotic Selenocysteine-specific Elongation Factor SelB
---
authors: 'Kubota, Mitsuhiro and Hagiya, Masami'
title: 'Minimum Basin Algorithm: An Effective Analysis Technique for DNA Energy Landscapes'
---
authors: 'Kudo, Mineichi and Somol, Petr and Pudil, Pavel and Shimbo, Masaru and Sklansky, Jack'
title: Comparison of Classifier-Specific Feature Selection Algorithms
---
authors: 'Rahmann, Sven and Mller, Tobias and Vingron, Martin'
title: On the Power of Profiles for Transcription Factor Binding Site Detection
---
authors: 'Ray, Debashish and Kazan, Hilal and Chan, Esther T and Castillo, Lourdes Pena and Chaudhry, Sidharth and Talukder, Shaheynoor and Blencowe, Benjamin J and Morris, Quaid and Hughes, Timothy R'
doi: 10.1038/nbt.1550
title: Rapid and systematic analysis of the RNA recognition specificities of RNA-binding proteins
---
authors: 'Rivas, Elena and Eddy, Sean R.'
title: Secondary structure alone is generally not statistically significant for the detection of noncoding RNAs
---
authors: 'Rocap, Gabrielle and Larimer, Frank W. and Lamerdin, Jane and Malfatti, Stephanie and Chain, Patrick and Ahlgren, Nathan A. and Arellano, Andrae and Coleman, Maureen and Hauser, Loren and Hess, Wolfgang R. and Johnson, Zackary I. and Land, Miriam and Lindell, Debbie and Post, Anton F. and Regala, Warren and Shah, Manesh and Shaw, Stephanie L. and Steglich, Claudia and Sullivan, Matthew B. and Ting, Claire S. and Tolonen, Andrew and Webb, Eric A. and Zinser, Erik R. and Chisholm, Sallie W.'
title: Genome divergence in two Prochlorococcus ecotypes reflects oceanic niche differentiation
---
authors: 'Sacha, Jarosław P.'
title: New Synthesis of Bayesian Network Classifiers and Cardiac SPECT Image Interpretation
---
authors: 'Saitou, Naruya and Nei, Masatoshi'
title: "The Neighbor-joining Method: A New Method for Reconstructing Phylogenetic Trees'"
---
authors: 'Schaal, Thomas D. and Maniatis, Tom'
title: 'Selection and Characterization of Pre-mRNA Splicing Enhancers: Identification of Novel SR Protein-Specific Enhancer Sequences'
---
authors: 'Skolnick, Jeffrey and Kolinski, Andrzej and Kihara, Daisuke and Betancourt, Marcos and Rotkiewicz, Piotr and Boniecki, Michal'
doi: 10.1002/prot.1172
title: 'Ab Initio Protein Structure Prediction via a Combination of Threading, Lattice Folding, Clustering, and Structure Refinement'
---
authors: 'Soinov, L. A.'
title: Supervised classification for gene network reconstruction
---
authors: 'Sommer, Ingolf and Zien, Alexander and von Ohsen, Niklas and Zimmer, Ralf and Lengauer, Thomas'
title: Confidence measures for protein fold recognition
---
authors: 'Sonnleitner, Elisabeth and Napetschnig, Johanna and Afonyushkin, Taras and Ecker, Karin and Vecerek, Branislav and Moll, Isabella and Kaberdin, Vladimir R. and Blasi, Udo'
doi: 10.1016/j.bbrc.2004.08.190
title: Functional effects of variants of the RNA chaperone Hfq
---
authors: 'Sonnleitner, Elisabeth and Schuster, Martin and Sorger-Domenigg, Theresa and Greenberg, Everett Peter and Bläsi, Udo'
doi: 10.1111/j.1365-2958.2005.05032.x
title: Hfq-dependent alterations of the transcriptome profile and effects on quorum sensing in Pseudomonas aeruginosa
---
authors: 'Stadler, Peter F. and Stephens, Christopher R.'
title: Landscapes and Effective Fitness
